skills/eval-analyze/scripts/discover.py

```
@@ -0,0 +1,139 @@
+    for pattern in patterns:
+        for path in sorted(glob(pattern)):
+            name = Path(path).parent.name
+            desc = ""
+            try:
+                with open(path) as f:
+                    content = f.read()
+                if content.startswith("---"):
+                    fm = yaml.safe_load(content.split("---")[1])
+                    desc = fm.get("description", "")[:80]
+            except Exception:
+                pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)